app/models/IndWhereDoYouLive.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.Aliases.Text
+import uk.gov.hmrc.govukfrontend.views.viewmodels.radios.RadioItem
+
+sealed trait IndWhereDoYouLive
+
+object IndWhereDoYouLive extends Enumerable.Implicits {
+
+  case object Wheredoyoulive extends WithName("where-do-you-live") with IndWhereDoYouLive
+  case object Wheredoyoulive2 extends WithName("where-do-you-live-2") with IndWhereDoYouLive
```

Comment:
Can this be given a more specific name to differentiate what both objects are doing?

Reply:
fixed